modules/kimberliteBidAdapter.js

```diff
@@ -17,18 +23,10 @@ const converter = ortbConverter({
 
   request(buildRequest, imps, bidderRequest, context) {
     const bidRequest = buildRequest(imps, bidderRequest, context);
-    bidRequest.site ||= {};
-    bidRequest.site.publisher ||= {
-      domain: bidderRequest.refererInfo.domain
-    };
-    bidRequest.site.page ||= bidderRequest.refererInfo.page;
+    deepSetValue(bidRequest, 'site.publisher.domain', bidderRequest.refererInfo.domain);
+    deepSetValue(bidRequest, 'site.page', bidderRequest.refererInfo.page);
+    deepSetValue(bidRequest, 'ext.prebid', VERSION_INFO);
```

Comment:
So if incoming ext.prebid exists this will also overwrite it!

```js
deepSetValue(bidRequest, 'ext.prebid.ver', VERSION_INFO.ver);
deepSetValue(bidRequest, 'ext.prebid.adapterVer', VERSION_INFO.adapterVer);
```

maybe is what you want?

Reply:
It was not intended that this field would be filled in by anyone, is that so?

Reply:
Anyway it's better to be changed in a way you provided.